cobalt/h5vcc/h5vcc_crash_log.h

```diff
@@ -59,6 +60,9 @@ class H5vccCrashLog : public script::Wrappable {
 
   void SetPersistentSettingWatchdogCrash(bool can_trigger_crash);
 
+  void ForceGarbageCollection(
```

Comment:
To be honest, i'm not super big fan of this as an extra API. There's a broader context around this, but we actually used to have a build config that was _always_ running with over-zealous garbage collection on for each and every test. See b/183953005

GC is configurable on V8 through runtime flags, and we can pass arguments to it at the boot time. Cobalt commandline flags also forward V8 config flags since a while.

We could actually just configure our CI to always run a job with GC agressively on in addition to current pipelines - for on-host testing that would be cheap. For on-device testing that would be pretty prohibitive though.

Reply:
Removed and restructured tests.